index.html

```diff
@@ -0,0 +1,186 @@
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(2).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Tonic</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+
+        <div class="card-container">
+            <img src="img/Snapshoot Portfolio(3).png" alt="Project Snapshoot" class="project-img">
+            <h2 class="card-tittle">Multi-Post Stories</h2>
+            <div class="frame2-img">
+                <img src="img/Frame 2.png" alt="">
+            </div>
+            <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="see-button">See Project</button>
+        </div>
+    </section>
+    
+    <!-- About section -->
+    <section class="about">
+        <h1 class="about-h1">About <br> Myself</h1>
+        <p class="info-summary">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h4 class="connect2">LET'S CONNECT</h4>
+
+        <ul id="social-icons2">
+            <li class="github">
+                <img src="icons/Git.png" alt="GitHub Icon">
+            </li>
+            <li class="linkedin">
+                <img src="icons/Linkedin.png" alt="LinkedIn Icon">
+            </li>
+            <li class="angel">
+                <img src="icons/AngelList.png" alt="AngelList Icon">
+            </li>
+            <li class="twitter">
+                <img src="icons/Twitter.png" alt="Twitter Icon">
+            </li>
+            <li class="medium">
+                <img src="icons/Medium.png" alt="Medium Icon">
+            </li>
+        </ul>
```

Comment:
[OPTIONAL]
- [ ] Nice work using a `ul` tag to group the media icons, and `li` per icon :clap: However it will be nice that each icon has a link (a tag). It will let you connect your social networks with the icons, using `href` attribute.